test/vastfiles/sample.xml

```diff
@@ -2,18 +2,18 @@
 <VAST version="2.1">
   <Ad id="ad_id_0001" sequence="1">
     <InLine>
-      <AdSystem version="2.0" ><![CDATA[AdServer]]></AdSystem>
+      <AdSystem version="2.0"><![CDATA[AdServer]]></AdSystem>
```

Comment:
question) are these modifications because of the string comparison?

Reply:
yes, string by reading file keeps that spaces, but xml.toStirng() is not.